init-tools.sh

```diff
@@ -68,6 +68,9 @@ get_current_linux_name() {
 }
 
 if [ -z "$__DOTNET_PKG" ]; then
+    if [ "$(uname -m | grep "i[3456]86")" = "i686" ]; then
```

Comment:
Ugh, I just realized that it should be comparing to != "" .. clearly this won't work for i386. Good enough for now.